content/collections/workflow/en/experiment-learnings.md

```diff
@@ -13,59 +13,113 @@ You’ve designed your experiment, rolled it out to your users, and given them e
 
 In the *Analysis* card, you’ll be able to tell at a glance whether your experiment has yielded **statistically-significant** results, as well as what those results actually are. Amplitude Experiment takes the information you gave it during the design and rollout phases and plugs them in for you automatically, so there’s no repetition of effort. It breaks the results out by variant, and provides you with a convenient, detailed tabular breakdown.
 
-{{partial:admonition type='note'}}
-This article continues directly from the [article in our Help Center on rolling out your experiment](/docs/feature-experiment/workflow/experiment-test). If you haven’t read that and followed the process it describes, do so before continuing here.
+Amplitude doesn't generate p-values or confidence intervals for experiments using binary metrics (for example, unique conversions) until each variant has 100 users **and** 25 conversions. Experiments using non-binary metrics need only to reach 100 users per variant.
+
+## Filter card
+
+On the Filter card, set criteria that updates the analysis on the page. Filter your experiment results with the following:
+
+* Date
+* Segment
+* Property
+
+### Date filter
+
+The date filter defaults to your experiment's start and end date. Adjust the range to scope experiment results to those specific dates.
+
+### Segment filter
+
+The segment filter enables you to select predefined segments, or create one ad-hoc. Predefined Experiment segments include:
```

Comment:
There are also segments for each of the targeting segments. There are also the OOTB segments "New users: Users who triggered at least one new user event during the selected date range.
Mobile web: Users who triggered events on a web platform from an Apple iPad, Apple iPhone, or Android device.
Desktop web: Users who triggered events on a web platform from a Mac or Windows device.
" that web analytics also uses https://amplitude.com/docs/analytics/web-analytics